src/ttkbootstrap/localization/msgs/pt.msg

```diff
@@ -1,82 +1,82 @@
-::msgcat::mcset pt_br "Themed"
-::msgcat::mcset pt_br "Standard"
-::msgcat::mcset pt_br "Current"
-::msgcat::mcset pt_br "New"
-::msgcat::mcset pt_br "Hue:"
-::msgcat::mcset pt_br "Sat:"
-::msgcat::mcset pt_br "Lum:"
-::msgcat::mcset pt_br "Hex:"
-::msgcat::mcset pt_br "Red:"
-::msgcat::mcset pt_br "Green:"
-::msgcat::mcset pt_br "Blue:"
-::msgcat::mcset pt_br "color dropper"
-::msgcat::mcset pt_br "Cancel"
-::msgcat::mcset pt_br "Search"
-::msgcat::mcset pt_br "Page"
-::msgcat::mcset pt_br "of"
-::msgcat::mcset pt_br "⎌"
-::msgcat::mcset pt_br "Reset table"
-::msgcat::mcset pt_br "Columns"
-::msgcat::mcset pt_br "Move"
-::msgcat::mcset pt_br "Align"
-::msgcat::mcset pt_br "Hide column"
-::msgcat::mcset pt_br "Delete column"
-::msgcat::mcset pt_br "Show All"
-::msgcat::mcset pt_br "Move to left"
-::msgcat::mcset pt_br "Move to right"
-::msgcat::mcset pt_br "Move to first"
-::msgcat::mcset pt_br "Move to last"
-::msgcat::mcset pt_br "Align left"
-::msgcat::mcset pt_br "Align center"
-::msgcat::mcset pt_br "Align right"
-::msgcat::mcset pt_br "Sort"
-::msgcat::mcset pt_br "Filter"
-::msgcat::mcset pt_br "Export"
-::msgcat::mcset pt_br "Delete selected rows"
-::msgcat::mcset pt_br "Sort Ascending"
-::msgcat::mcset pt_br "Sort Descending"
-::msgcat::mcset pt_br "Clear filters"
-::msgcat::mcset pt_br "Filter by cell's value"
-::msgcat::mcset pt_br "Hide select rows"
-::msgcat::mcset pt_br "Show only select rows"
-::msgcat::mcset pt_br "Export all records"
-::msgcat::mcset pt_br "Export current page"
-::msgcat::mcset pt_br "Export current selection"
-::msgcat::mcset pt_br "Export records in filter"
-::msgcat::mcset pt_br "Move up"
-::msgcat::mcset pt_br "Move down"
-::msgcat::mcset pt_br "Move to top"
-::msgcat::mcset pt_br "Move to bottom"
+::msgcat::mcset pt "Ok"
+::msgcat::mcset pt "Retry"
+::msgcat::mcset pt "Delete"
+::msgcat::mcset pt "Next"
+::msgcat::mcset pt "Prev"
+::msgcat::mcset pt "Yes"
+::msgcat::mcset pt "No"
+::msgcat::mcset pt "Open"
+::msgcat::mcset pt "Close"
+::msgcat::mcset pt "Add"
+::msgcat::mcset pt "Remove"
+::msgcat::mcset pt "Submit"
+::msgcat::mcset pt "Family"
+::msgcat::mcset pt "Weight"
+::msgcat::mcset pt "Slant"
+::msgcat::mcset pt "Effects"
+::msgcat::mcset pt "Preview"
+::msgcat::mcset pt "Size"
+::msgcat::mcset pt "Should be of data type"
+::msgcat::mcset pt "Invalid data type"
+::msgcat::mcset pt "Number cannot be greater than"
+::msgcat::mcset pt "Out of range"
+::msgcat::mcset pt "Previous"
+::msgcat::mcset pt "The quick brown fox jumps over the lazy dog."
+::msgcat::mcset pt "Font Selector"
+::msgcat::mcset pt "normal"
+::msgcat::mcset pt "bold"  "negrito"
+::msgcat::mcset pt "roman"
+::msgcat::mcset pt "italic"
+::msgcat::mcset pt "underline"
+::msgcat::mcset pt "overstrike"
+::msgcat::mcset pt "Color Chooser"
+::msgcat::mcset pt "Advanced"
+::msgcat::mcset pt "Themed"
+::msgcat::mcset pt "Standard"
+::msgcat::mcset pt "Current"
+::msgcat::mcset pt "New"
+::msgcat::mcset pt "Hue:"
```

Comment:
Per my other comment, if we use the f-string in the translation, then we can use the token here without adding the colon. This would allow us to use these tokens with or without the colon.